src/test/java/com/rocketden/main/service/RoomServiceTests.java

```diff
@@ -172,20 +172,14 @@ public void getRoomFailure() {
 
     @Test
     public void sendSocketUpdate() {
-        User user = new User();
-        user.setNickname("test");
-        Set<User> users = new HashSet<>();
-        users.add(user);
-
-        UserDto userDto = new UserDto();
-        userDto.setNickname("test");
-        Set<UserDto> expected = new HashSet<>();
-        expected.add(userDto);
+        RoomDto roomDto = new RoomDto();
+        roomDto.setHost(new UserDto());
```

Comment:
Should you confirm that any `UserDto` nickname that is provided as well is also checked?

Reply:
Yeah good point, just updated!